lib/file.js

```diff
@@ -119,6 +126,26 @@ client.uploadFile = function(userId, fileURI, options) {
   }
 };
 
+client.processQueue = function() {
```

Comment:
Yes, the intention is to call this function from the app.js or where ever the developer would prefer. There is also an expectation the developer will handle offline and online events, saving and processing the queue as needed. 

I will update README detailing this. 

What do you think